db/migrate/20220531075527_update_bolt_configuration_environment_column_restrictions.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateBoltConfigurationEnvironmentColumnRestrictions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :solidus_bolt_bolt_configurations, :environment, :integer, null: false, default: 1
+    remove_column :solidus_bolt_bolt_configurations, :environment, :integer
```

Comment:
This migration doesn't work.

```
Would you like to run the migrations now? [Y/n]
         run  bin/rails db:migrate from "."
== 20220531105410 UpdateBoltConfigurationEnvironmentColumnRestrictions: migrating
-- add_column(:solidus_bolt_bolt_configurations, :environment, :integer, {:null=>false, :default=>1})
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

SQLite3::SQLException: duplicate column name: environment
/Users/danielepalombo/Code/solidus_bolt/sandbox/db/migrate/20220531105410_update_bolt_configuration_environment_column_restrictions.solidus_bolt.rb:4:in `change'
```

Probably it happens because the column is added before its removal.

In order to avoid to remove/adding a column and losing the data, what do you think to use the two method `up` and `down` changing just the column definition?

Reply:
Modified here [c216f487](https://github.com/nebulab/solidus_bolt/commit/c216f4871c2a755c7c66f5930fa793e25e37d7e0)

I tried using the `up` and `down` methods but it didn't work as expected.

I have used the `change` method for now, which is working correctly

Reply:
What do you mean "it doesn't work"? Can you provide more details? Up/down is the standard for the migration and it should work as expected